backend/main.py

```diff
@@ -1,10 +1,29 @@
-from logging import DEBUG, INFO, basicConfig, getLogger
-from os import getenv
+# Full imports
+import aiohttp_cors
 
+# Partial imports
 from aiohttp import ClientSession
+from aiohttp.web import Application, run_app, static
+from aiohttp_jinja2 import setup as jinja_setup
+from asyncio import get_event_loop, sleep
+from json import dumps, loads
+from logging import DEBUG, INFO, basicConfig, getLogger
+from os import getenv, path
+from subprocess import call
+
+# local modules
+from browser import PluginBrowser
+from helpers import get_user, get_user_group
+from injector import inject_to_tab, tab_has_global_var
+from loader import Loader
+from updater import Updater
+from utilities import Utilities
 
+USER = get_user()
+GROUP = get_user_group()
+HOME_PATH = "/home/"+USER
```

Comment:
```suggestion
HOME_PATH = os.path.expanduser("~")
```
It is possible to have a home directory outside /home

Reply:
While this is technically correct, we are focused on supporting primarily Steam Deck hardware + SteamOS 3. We will take steps to help support other distros but if their home directories do not exist withing ``/home`` then that's going to be exotic enough that we're not going to spend time supporting that. Apologies.